homeassistant/components/reolink/select.py

```diff
@@ -72,6 +72,16 @@ class ReolinkSelectEntityDescription(
         supported=lambda api, ch: api.supported(ch, "ptz_presets"),
         method=lambda api, ch, name: api.set_ptz_command(ch, preset=name),
     ),
+    ReolinkSelectEntityDescription(
+        key="play_quick_reply_message",
+        translation_key="play_quick_reply_message",
+        icon="mdi:message-reply-text-outline",
+        get_options=lambda api, ch: list(api.quick_reply_dict(ch).values())[1:],
+        supported=lambda api, ch: api.supported(ch, "play_quick_reply"),
+        method=lambda api, ch, mess: api.play_quick_reply(
+            ch, file_id=[k for k, v in api.quick_reply_dict(ch).items() if v == mess][0]
+        ),
```

Comment:
Multiline lambdas are not wanter, make a function instead.

Reply:
A few lines below (94-96) a practically identical lambda is already present:
```
        method=lambda api, ch, mess: api.set_quick_reply(
            ch, file_id=[k for k, v in api.quick_reply_dict(ch).items() if v == mess][0]
        ),
```

Having it like this keeps everything of one entity together instead off having seperate small functions somewhere else in the file.

Reply:
I've been told in multiple reviews by Martin to not have multi line lambdas so I'm just sharing my experience

Reply:
The other multi line lambda was approved by Frenck in this PR https://github.com/home-assistant/core/pull/89656

Will wait for Frenck/Martin too see what they think about it.

Reply:
Suggestion: Break out `[k for k, v in api.quick_reply_dict(ch).items() if v == mess][0]` into a function as there is an other entity description which uses this. Than you could wrap your lambda into parentheses, this would make it more readable:

```python
ReolinkSelectEntityDescription(
    key="play_quick_reply_message",
    translation_key="play_quick_reply_message",
    icon="mdi:message-reply-text-outline",
    get_options=lambda api, ch: list(api.quick_reply_dict(ch).values())[1:],
    supported=lambda api, ch: api.supported(ch, "play_quick_reply"),
    method=lambda api, ch, mess: (
        api.play_quick_reply(ch, file_id=_get_quick_reply)
    ),
),
```

Reply:
@jpbede thanks for the suggestion, that indeed makes it more readible.